src/context.rs

```diff
@@ -107,6 +107,9 @@ impl Game {
         let ppu = Ppu::new(cgb_mode);
         let ppu_mem = Rc::new(RefCell::new(ppu.memory()));
         let ppu_reg = Rc::new(RefCell::new(ppu.registers()));
+        if !cfg!(feature = "bios") {
```

Comment:
Instead of using an if statement you can use `#[cfg(not(feature="bios"))]` above the line that should be enabled when the bios feature is disabled.